src/converter/linkml_converter.py

```diff
@@ -1,15 +1,23 @@
-from src.converter.converter import SnakemakeConverterTrait
-from src.helpers import merge_dict_list
+import os.path
 
+import yaml
 
-class LinkMLConverter(SnakemakeConverterTrait):
-    def __init__(self, benchmark):
+from src.converter.converter import ConverterTrait
+from src.utils.helpers import merge_dict_list, load_yaml
+
+
+class LinkMLConverter(ConverterTrait):
+    def __init__(self, benchmark_file):
         super().__init__()
-        self.benchmark = benchmark
+        self.benchmark_file = os.path.abspath(benchmark_file)
```

Comment:
__Fix(code duplication)__: If you assign something as attribute, there is no reason to also provide a method that calls the attribute. Maybe this comes from the `Converter` class, but why not defining it as attribute there already.

Reply:
Fair point, the main reason was overloading, and I was just following the pattern, even though the attribute was already declared in the child class. Regardless, I think most of this code will be simplified once we remove the `YamlConverter` and get rid of the unnecessary hierarchy. I could do this already, but I don't want to overlap with your refactoring?